rewrite-java/src/main/java/org/openrewrite/java/ChangeType.java

```diff
@@ -505,6 +511,18 @@ private Expression updateOuterClassTypes(Expression typeTree) {
         private boolean isTargetFullyQualifiedType(JavaType.@Nullable FullyQualified fq) {
             return fq != null && TypeUtils.isOfClassType(fq, originalType.getFullyQualifiedName()) && targetType instanceof JavaType.FullyQualified;
         }
+
+        private boolean becomesAmbiguous(JavaSourceFile cu) {
```

Comment:
This once again gave me warnings when opened in the IDE; please keep that `F2` before commit in mind, and 
![image](https://github.com/user-attachments/assets/936d60c0-17f7-4bb2-ba54-d4db8e1106b2)

Also be sure to enable this option for yet another nudge to not push any code with new warnings; that helps me review & improve code quality.
![image](https://github.com/user-attachments/assets/e68a2986-c208-4247-aa22-34301a567262)